README.md

```diff
@@ -60,19 +59,20 @@ and disabled by default:
 
 ## Platforms supported
 
-CI means whether we have an automatic CI for this platform. If someone has CI for these platforms or any other platforms not listed here and would like to add them to our automatic CI, please get in touch with us, it will much appreciated!
-
-| Platform  | Support | CI |
-| ------------- | ------------- |----|
-| Linux x64  | &#10003;  | &#10003; |
-| Linux x86  | &#10003;  | &#10003; |
-| Linux ARM64  | &#10003; | &#10003; |
-| Windows x86  | &#10003;  | &#10003; |
-| Windows x64  | &#10003;  | &#10003; |
-| Windows arm64  | &#10003;  | x |
-| macOS x64  | &#10003;  | &#10003; |
-| macOS M1  | &#10003;  | x |
-| S390  | &#10003;  | x |
+CI means whether we have an automatic CI for this platform. If someone has CI for these platforms or any other platforms not listed here and would like to add them to our automatic CI, please get in touch with us, it will be much appreciated!
+
+| Platform      | Support | CI |
+|---------------|---------|----|
```

Comment:
can you use `:--:` for support & CI to center align the contents ?

Reply:
Ah, nice, didn't know that.